stage1/init/common/units.go

```diff
@@ -328,6 +328,16 @@ func (uw *UnitWriter) SetupAppIO(p *stage1commontypes.Pod, ra *schema.RuntimeApp
 			if ok {
 				file.WriteString(fmt.Sprintf("STAGE1_LOGMODE=%s\n", logMode))
 			}
+			switch logMode {
+			case "k8s-plain":
+				kubernetesLogPath, ok := ra.Annotations.Get("coreos.com/rkt/experiment/kubernetes-log-path")
```

Comment:
Can we make this `kubernetes-log-filename`

Reply:
`log path` is the term that CRI and kubelet use and I just wanted to be consistent. But I don't care if you seriously care about it. ;)

Reply:
Right. In the CRI they use `LogDirectory` and `LogPath`, which I find confusing. Since these annotations are k8s-specific I guess staying consistent with their names is ok. Disregard the comment then.